.github/tools/release_linux.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
+set -e -x
+
+# Remove the now private ppa. This can be removed after the docker image removes the
+# pre-installed python packages from this ppa.
+rm -f /etc/apt/sources.list.d/jonathonf-ubuntu-python-3_6-xenial.list
+
+ln -sf /usr/bin/python3.5 /usr/bin/python3 # Py36 has issues with add-apt
```

Comment:
This is all magic adapted from https://github.com/tensorflow/addons/blob/master/tools/ci_build/builds/release_linux.sh